superset-frontend/spec/javascripts/sqllab/actions/sqlLab_spec.js

```diff
@@ -175,15 +166,6 @@ describe('async actions', () => {
       });
     });
 
-    it.skip('parses large number result without losing precision', () =>
```

Comment:
Same here, would rather leave it skipped. Fixing the test is the better way to remove the warning :D